interview-questions.md

```diff
@@ -6,42 +6,42 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-Type your initials here to confirm you read the instructions:
+Type your initials here to confirm you read the instructions: IV
 
 1. What is object-oriented programming? How is it different than functional programming?
 
-Your answer:
+Your answer: Object-oriented programming is the focus of using objects to code. For example, Ruby is a coding language that uses object-oriented programming to function. It is different than functional programming because object-oriented programming uses objects as well as less functionality. In other words, functional programming uses more functions and functionality. Ex: In JavaSciript, in order to capitalize a string in an array, which uses functional programming, you need to find the index of the word hou wish to capitalize and you use toUpperCase. While object-oriented programming allows you to call on the string you want to capitalize and use .capitalize
 
-Researched answer:
+Researched answer: A style of programming with the use of objects closely linked with methods. Object-oriented programming uses objects and methods while functional programming uses functional factors like variables and functions. 
 
 2. Ruby has an implicit return. What does that mean?
 
-Your answer:
+Your answer: I am not sure what an implicit return means. 
 
-Researched answer:
+Researched answer: An implicit return means that a function is returned without using the word return. Ruby uses p in methods instead of return. 
```

Comment:
In Ruby, `p` will print something to the console, it's important to understand that it is not the same as a return.